docs/source/Plugin/P146.rst

```diff
@@ -79,9 +79,11 @@ Available options:
 
 ..
 
-* **Join Samples with same Timestamp**: When enabled will join/combine data with the same timestamp.
+* **Join Samples with same Timestamp**: When enabled will join/combine data with the same timestamp into a single CSV file.
 
-* **Export only set tasks**:
+* **Export only enabled tasks**: Export only the tasks that are enabled, limiting the amount of data stored and sent.
```

Comment:
The data is already stored.
So it is just about reducing the data sent by the Cache Reader plugin.

For binary data this will only change the 'taskinfo' string being sent.
For CSV formatted data the data reduction may be significant as not enabled task columns will not have to be sent.